.gitignore

```diff
@@ -2,3 +2,5 @@ env.sh
 .mypy_cache
 notebooks/output
 notebooks/repos
+.venv/
```

Comment:
For those I usually prefer .git/info/exclude or a global .gitignore since it depends on specific environments but i don't mind including it if you prefer.